server/test/unit/server/agent/direct/test_agent.py

```diff
@@ -44,14 +59,16 @@ def test_delete_queue(self, queue):
         agent.delete_queue(url, name)
 
         # validation
-        queue.assert_called_once_with(name)
+        add_connector.assert_called_once_with()
+        queue.assert_called_once_with(name, url)
         queue = queue.return_value
```

Comment:
Consider avoiding variable redefinition like this. Instead of storing this your lines 65 and 66 could become:

```
queue.return_value.purge.assert_called_once_with()
queue.return_value.delete.assert_called_once_with()
```